### PR TITLE
HHH-12858 - integration overrides during JPA bootstrap ought to overr…

### PR DESCRIPTION
**Pull request by dreab8**
…ide all logically related settings

### CODE REVIEW

File: hibernate-core/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java [hunk 1]
```diff
@@ -218,16 +218,14 @@ private EntityManagerFactoryBuilderImpl(
 		ssrBuilder.applySettings( configurationValues );
 
 		this.standardServiceRegistry = ssrBuilder.build();
-		configure( standardServiceRegistry, mergedSettings );
+
+		configureIdentifierGenerators( standardServiceRegistry );
```
Comment (sebersole) [inline hibernate-core/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java]: Nice change, +1

File: hibernate-core/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java [hunk 2]
```diff
@@ -1092,10 +1060,7 @@ private void configure(StandardServiceRegistry ssr, MergedSettings mergedSetting
 	}
 
 	@SuppressWarnings("unchecked")
-	protected List<AttributeConverterDefinition> populate(
-			MetadataSources metadataSources,
-			MergedSettings mergedSettings,
-			StandardServiceRegistry ssr) {
+	protected List<AttributeConverterDefinition> populate(MetadataSources metadataSources) {
```
Comment (sebersole) [inline hibernate-core/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java]: I'm going to send a PR for some minor additional changes here.

**Comment by sebersole** [comment]
Is this something we still need to integrate?

**Comment by dreab8** [comment]
you integrated it

**Pull request closed without merging**